blog-cases/2022-06-24-equiptalk.md

```diff
@@ -1,12 +1,12 @@
-# How EquipTalk Empowers Launch Vehicles with Shifu
+# How EquipTalk Empowers Rockets with Shifu
 
 EquipTalk Ltd. is dedicated to develope software-defined high-end equipments, use aerospace measurement technology to build the next-generation digital base for high-end equipments, build the mapping engine of the virtual and real world of high-end equipment, and empower equipment in all aspects. 
```

Comment:
EquipTalk Ltd. is dedicated to develop software-defined high-end equipments. Leveraging the aerospace measurement technology as the next-generation digital foundation for high-end equipments, EquipTalk strives to empower high-end equipment in all aspects.